.goreleaser.yaml

```diff
@@ -5,7 +5,7 @@ before:
     - go mod tidy
 release:
   # Mark nightly build as prerelease based on tag
-  prerelease: '{{ contains .Tag "-nightly" }}'
+  prerelease: auto
```

Comment:
What's this for?

Reply:
Earlier expression was not evaluating correctly and nightly releases were not marked as prerelease. So this is another way of achieving the same.

Reply:
Let's do this in another PR.

Reply:
Done